src/Pillar-Tests-Project/ProjectDefaultTargetTest.class.st

```diff
@@ -0,0 +1,60 @@
+Class {
+	#name : #ProjectDefaultTargetTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'project'
+	],
+	#category : #'Pillar-Tests-Project'
+}
+
+{ #category : #'as yet unclassified' }
+ProjectDefaultTargetTest >> configHTMLContent [
```

Comment:
This should be categorized

Reply:
Categorized in _tests_ (as in Pillar-Tests-ExportHTML)